microsoft-edge/devtools-guide-chromium/css/index.md

```diff
@@ -156,27 +156,29 @@ Using separate layouts allows for a one-column layout for mobile devices and mul
 
 Use the following steps to debug or test the media queries that you defined in your CSS.
 
-1. To open DevTools, you can right-click in a webpage and then select **Inspect**.
+1. Open the [CSS Examples](https://microsoftedge.github.io/Demos/devtools-css-get-started/) demo page in a new window or tab.
+
+1. To open DevTools, right-click in the webpage and then select **Inspect**.
 
 1. Click the **Toggle device emulation** (![Device Emulation icon](./index-images/device-emulation-icon-light-theme.png)) button.  Or, when DevTools has focus, press **Ctrl+Shift+M** (Windows, Linux) or **Command+Shift+M** (macOS).
 
-   <!-- todo: update to show new tooltip: -->
+   The Device Toolbar opens in the webpage, and the webpage is now rendered within the Device Emulation pane:
```

Comment:
```suggestion
   The device toolbar opens in the webpage, and the webpage is now rendered within the Device Emulation pane:
```
* consistent casing.  d/k alt-text, but rendered text only has this 1 hit, uppercase.